pkg/util/trace/impl/motrace/report_statement.go

```diff
@@ -108,7 +108,9 @@ func (s *StatementInfo) FillRow(ctx context.Context, row *table.Row) {
 	s.exported = true
 	row.Reset()
 	row.SetColumnVal(stmtIDCol, uuid.UUID(s.StatementID).String())
-	row.SetColumnVal(txnIDCol, uuid.UUID(s.TransactionID).String())
+	if !s.IsZeroTxnID() {
```

Comment:
fix TransactionID with default value `""`, instead of using `00000000-0000-0000-0000-000000000000` (for MOCloud)